package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/open-rpc-docs-react",
-  "version": "0.1.2",
+  "version": "1.1.2",
```

Comment:
no need to change the version here, CI will do it when we go to release

Reply:
```suggestion
  "version": "0.1.2",
```

Reply:
@shanejonas Got it! Fixed in the last commit.